src/IssueViz.Security.UnitTests/Taint/TaintList/TaintIssuesControlViewModelTests.cs

```diff
@@ -882,7 +876,8 @@ public void OnIssueVizPropertyChanged_SuppressedProperty_FromSuppressedToNotSupp
             IShowInBrowserService showInBrowserService = null,
             IIssueSelectionService selectionService = null,
             IMenuCommandService menuCommandService = null,
-            ISonarQubeService sonarQubeService = null)
+            ISonarQubeService sonarQubeService = null,
+            IThreadHandling threadHandling = null)
```

Comment:
Miscellaneous: add `threadHandling` to the test so we don't use the real class by default.